packages/ng/callout/src/lib/callout/callout.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="callout palette-{{palette}}" [class.mod-S]="size === 's'" [class.mod-tiny]="tiny">
+	<div class="callout-icon" *ngIf="icon">
+		<span aria-hidden="true" class="lucca-icon icon-{{icon}}"></span>
+	</div>
+	<div class="callout-content">
+		<strong class="callout-content-title" *ngIf="title">{{title}}</strong>
+		<div class="callout-content-description">
+			<ng-content></ng-content>
+		</div>
+	</div>
+	<button *ngIf="removable" type="button" class="callout-kill" (click)="hide()"></button>
```

Comment:
The button is missing a masked text for accessibility.

```suggestion
	<button *ngIf="removable" type="button" class="callout-kill" (click)="hide()">
	    <span class="u-mask">{{ A11Y_DISMISS_CALLOUT | translate}}</span>
	</button>
```

Reply:
While I agree on that, I think it raises a larger question about i18n in a design system. Here we'd be using `ngx-translate`, some apps are using `transloco`, and this is, in the end, more of an app-dependant decision that shouldn't be handled here (like, how are we going to implement it in the design system at all? Does that mean that we'd have to provide json files for i18n and ask the consumer app to host them?).

There's a lot of different approaches we could discuss on that, like using an input for a11y label or providing a global i18n configuration object, or even providing an i18n translation approach from a configuration object given in `forRoot` or something.

Reply:
Don't some ng-components already include translations that we use in apps with different i18n setups?

My point was more that the button needs a text alternative to the icon for accessibility purposes. This could be passed as an input as well if we don't want the component to include a translations file.

Reply:
We have no translated values in lucca-front at the moment ^^.

Providing an alternative for the icon is for sure something we should provide, that's not the point that I'm trying to solve here.

My point is more that if we want proper a11y, we want this translated for sure, and having the library consumer provide the alt text doesn't really make sense as they shouldn't have to provide the alt text for something they didn't even add themselves (it'd mean having something like a `closeAltLabel` input that's used for the label).

Which is why you're raising a very interesting and complex problem that wed'll have to solve for the future too, by chosing which approach we want to use and stick to it for the new components (not to mention button itself will become a component, well a directive to be accurate, with its own alt input to make this easier).

Reply:
I agree, the alt text should be contained in the component.

There are some translated values here:

https://github.com/LuccaSA/lucca-front/blob/rc/packages/ng/establishment/src/lib/select/select-all/establishment-select-all.component.html

Reply:
Oh, I see why I didn't find them, I was looking for `translate` or `transloco` usages in html files.

I'm not sure that's the best way to implement translation for the design system tbh, this one just uses a const exported from another file here: https://github.com/LuccaSA/lucca-front/blob/rc/packages/ng/establishment/src/lib/select/select-all/establishment-select-all.translate.ts which raises a maintenance issue.

I'm going to merge this PR regardless of that because the close mode isn't used in many places for now and I'll add a TODO to make sure it's not forgotten, but that topic is very interesting and will have to be solved for sure, we've got some ideas on how to make it as flexible as possible for everyone to use properly 😄

Reply:
The current translation system is agnostic: it is the simplest way but it is not satisfying (all languages are bundled, we do not support variables nor plural, we dont have tools to ensure all translations exist in all languages).
Waiting for a translation refactor, you can stick with the current system:
- one file that declare a Record, an Interface and an InjectionToken ([example](https://github.com/LuccaSA/lucca-front/blob/63d46ba76e6433e07b672552395126dc848b88bc/packages/ng/modal/src/lib/modal.translate.ts))
- inject correct translations using `getIntl` ([example](https://github.com/LuccaSA/lucca-front/blob/6a13bbc422ea009679d54ff70e7ec5ae45100195/packages/ng/modal/src/lib/modal-panel.component.ts#L22))